cmds/ps/ps.go

```diff
@@ -168,17 +174,61 @@ func (pT *ProcessTable) PrepareString() {
 	pT.fstring = fstring
 }
 
+func mapSubset(check map[byte]int, ref map[byte]int) bool {
+	for cLet, cNum := range check {
+		if rNum, ok := ref[cLet]; ok {
+			if cNum != rNum {
+				return false
+			}
+		} else {
+			return false
+		}
+	}
+	return true
+}
+
+func isPermutation(check string, ref string) bool {
```

Comment:
This seems more complicated than it needs to be.

```go
func isAnagram(arg string, ref string) bool {
  // The canonical way to create a set in Go is to use an empty struct{}, as its len is 0.
  argMap := make(map[rune]struct{})

  for _, letter := range arg {
    argMap[letter] = struct{}{}
  }

  for _, letter := range ref {
     if _, ok := argMap[letter]; !ok {
       return false
     }
  }
  return true
}
```

(note that it doesn't handle duplicate letters, but it doesn't need to for our use case)